src/app/components/MediaLoader/configs/clipMedia.ts

```diff
@@ -106,7 +108,7 @@ export default ({
       },
       statsObject: {
         ...basePlayerConfig.statsObject,
-        clipPID: versionID,
+        clipPID: videoId,
```

Comment:
Looking at the SMP/ Cookbook docs I see that this should be the pid and not the vpid (version pid). 

I have updated this here and in tests.